mycroft/tts/__init__.py

```diff
@@ -297,8 +297,10 @@ def execute(self, sentence, ident=None):
         create_signal("isSpeaking")
         if self.phonetic_spelling:
             for word in re.findall(r"[\w']+", sentence):
-                if word in self.spellings:
-                    sentence = sentence.replace(word, self.spellings[word])
+                if word.lower() in self.spellings:
+                    sentence = sentence.replace(word,
+                                                self.spellings[word.lower()])
+                    LOG.info(sentence)
```

Comment:
If we keep the new log here, do we want to add extra into like `New phonetic sentence:`?

Reply:
Sorry that's not supposed to be there. will remove.